storage-inventory/quick-start/README.md

```diff
@@ -0,0 +1,108 @@
+# Storage Inventory Deployment Quick Start
+
+## Introduction
+This document is meant to guide you through a simple deployment of Storage Inventory site services and applications and  to introduce some of the concepts required.  Please also look at the development documentation -- [opencadc/storage-inventory](https://github.com/opencadc/storage-inventory), especially the [architecture description](https://github.com/opencadc/storage-inventory/tree/master/storage-inventory-dm).
+
+### Assumptions in this document
+- We're not deploying a production instance of the services -- examples presented here only show simple docker container deployments without orchestration (e.g. Docker Swarm, kubernetes).  
+- There are dependencies on ancillary CADC services.  These depenedencies are necessary if the Storage Site being deployed is meant to be part of the CADC/CANFAR ecosystem, but not if this is meant to be an independent installation.  This document does not describe how to deploy those ancillary services.
+
+## Basic Concepts
+### Storage site vs Global site
+- a _Storage site_ maintains an inventory of and provides access to the files stored at a particular location.  If you have files in multiple data centres, or more than one storage platform in one data centre (e.g. some files on a posix file-system and some on Ceph object storage), you would have more than one _Storage site_.  
+
+  A _Storage site_ runs two services:
+  - [`minoc`](minoc/README.md) - file service. Provides a file upload and download service.  
+  - [`luskan`](luskan/README.md) - metadata query service. Provides a service for querying the inventory database using the TAP protocol.  
+  
+  A _Storage site_ will also need to run applications to validate the inventory contents:
+  - [`tantar`](tantar/README.md) - file validation.  This compares the artifact metadata in the inventory database with the actual storage, ensuring that they are in sync.  
+  
+  Additional applications will be needed if the _Storage site_ is meant to be synced with other _Storage sites_, which will also require a _Global site_ to be deployed.
+  - `fenwick` - incremental metadata synchronization between the _Storage site_ and _Global site_.  Only compares artifacts since the last run.  
+  - `ratik` - full metadata validation between the _Storage site_ and _Global site_.
+  - `ringhold` - metadata validation after a site policy change.  Subtle difference with above.
```

Comment:
ringhold removes a namespace; that's not a validation operation, just a quicker way to do removal of a subset